tests/modules/programs/nix-index/integrations.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs, ... }:
+let
+  fishRegex = ''
+    function __fish_command_not_found_handler --on-event fish_command_not_found
+        /nix/store/.*command-not-found $argv
+    end
+  '';
+in {
+  config = {
+    programs.bash.enable = true;
+    programs.fish.enable = true;
+    programs.zsh.enable = true;
+
+    programs.nix-index.enable = true;
```

Comment:
Wouldn't this also need overlays?

Reply:
Yeah, I guess should have at least
```
    nixpkgs.overlays = [
      (self: super: {
        zsh = pkgs.writeScriptBin "dummy-zsh" "";
      })
    ];
```

The fish tests don't seem to have any overlay so I don't know what kind of overlay would be necessary for those. I think a zsh overlay is sufficient for now.

Reply:
What is the use for those overlays?

Reply:
The point is so that the CI test runner doesn't have to download all of the dependencies.